build/webpack.karma.config.js

```diff
@@ -1,17 +1,13 @@
 require('babel-polyfill');
 
-const { argv } = process;
-const isDebug = argv.find(arg => {
-    return arg === '--auto-watch' || arg === '--no-single-run';
-});
-
 const { IgnorePlugin } = require('webpack');
 const commonConfig = require('./webpack.common.config');
 
 const baseConfig = commonConfig('en-US');
 
 const config = {
     ...baseConfig,
+    devtool: 'inline-source-map',
```

Comment:
Is this needed as part of this change? Does it affect the production bundle?

Reply:
> Is this needed as part of this change? Does it affect the production bundle?

The next line specifies the mode to be 'development', so it does not affect the production bundle. This only makes the testing slower but it solves enzyme preprocess problem.